FirebaseDatabase/Sources/Realtime/FWebSocketConnection.m

```diff
@@ -292,9 +409,42 @@ - (void)webSocket:(FSRWebSocket *)webSocket
     [self onClosed];
 }
 
+#endif // TARGET_OS_WATCH
+
 #pragma mark -
 #pragma mark Private methods
 
+#if TARGET_OS_WATCH
+
+/// Sends data to the web socket through native APIs.
```

Comment:
The rest of this code base generally seems to use JavaDoc for method comments.

Reply:
Will change, thanks